### PR TITLE
requirements documentation

### PR DESCRIPTION
**Pull request by ahwagner**
Initialize sphinx documentation for project, provide overview of requirements designated in #1, #3, and (partially) #4.

### CODE REVIEW

File: docs/make.bat [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```
Comment (bwalsh) [inline docs/make.bat]: Should we have a linux make file? That way anyone can build or eventually we could set up a continuous build ...

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'metakb'
+copyright = '2018, Alex Wagner'
```
[the remark this answers is not in the document]
Comment (ahwagner) [reply docs/conf.py]: Completely agree. This was auto-generated by Sphinx, but think that "Variant Interpretation for Cancer Consortium" is more appropriate value for copyright statement. License covers terms of reuse under the copyright. Will address in next commit.

File: docs/make.bat (hunk 1 above)
Comment (ahwagner) [reply docs/make.bat]: Linux Makefile is at /docs/Makefile.

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+author = 'Alex Wagner'
+
+# The short X.Y version
+version = ''
```
[the remark this answers is not in the document]
Comment (ahwagner) [reply docs/conf.py]: Okay. Not strictly needed without any functionality, but it doesn't hurt to set up. Will address in subsequent commit.

**Comment by ahwagner** [comment]
Thanks for pointing that out, @bwalsh. Still working out the kinks, but now you (and the other metakb-dev members) should have read/write access to the repo, fixing this issue.

**Pull request merged**